### DISCUSSION TITLE
节点多就卡顿

### DISCUSSION THREAD

**Discussion by zmcode** [antvis/X6#2382] [Q&A]
目前所有节点都是html的形式, 因为需要使用到图片和文本, 现在67个节点,初次渲染大概就需要2秒钟, 切换布局的时候也是需要等2 3秒, 体验很不友好, 我尝试加到一百多个, 就更加卡了

我这边的需求大概是需要能自动切换布局, 斥力布局、树状布局， 环形布局等等， 每次切换的时候，我都会调用graph.clearCells(), 然后在addNode 和 addEdge, 是因为没有用fromJSON的原因导致的吗?

![image](https://user-images.githubusercontent.com/48686959/179931863-2da11822-56d8-48b7-a8fe-3d003404cc0a.png)
![image](https://user-images.githubusercontent.com/48686959/179931964-7baab819-9fe3-40c1-a35c-12ccdf74a2e3.png)

**Comment by NewByVector** [comment]
不要一个一个的 addNode，可以批量 addNodes。

**Comment by wb-wenbei** [comment]
G6了解一下
